.ci/install_linux.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-set -e
+set -ex
+
+cd "${HOME}/workspace"
 
 if [ ${TRAVIS} ]; then
   AIKIDO_BUILD_DIR="${TRAVIS_BUILD_DIR}"
```

Comment:
Since we're only using Travis, I think we can delete this `if` statement and just use `${TRAVIS_BUILD_DIR}` in line 13.